res/res_pjsip_pubsub.c

```diff
@@ -4085,25 +4124,32 @@ static void pubsub_on_rx_refresh(pjsip_evsub *evsub, pjsip_rx_data *rdata,
 		sub_tree->state = SIP_SUB_TREE_TERMINATE_PENDING;
 	}
 
+	endpoint = ast_pjsip_rdata_get_endpoint(rdata);
```

Comment:
handler also should directly be accessible, and guaranteed to be non-NULL if a refresh